File: Selection statement/Qusetion1/question1.java
import java.util.Scanner;

public class question1 {
    static void EverOrOdd() {
        int n;

        try (Scanner scan = new Scanner(System.in)) {
            System.out.println("Enter a number");
            n = scan.nextInt();
        }

        if (n % 2 == 0) {
            System.out.println(n + " is Even");
        } else {
            System.out.println(n + " is Odd");
        }
    }

    public static void main(String[] args) {
        EverOrOdd();
    }
}

File: Selection statement/Qusetion1/question1.c
#include <stdio.h>

int EvenOrOdd(int n)
{
    if (n % 2 == 0)
    {
        printf("%d is even\n", n);
    }
    else
    {
        printf("%d is odd\n", n);
    }
}

int main()
{
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    EvenOrOdd(n);
    return 0;
}